### DISCUSSION TITLE
Do we still need useAtomCallback?

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#1757] [Q&A] [answered]
Why would I use `useAtomCallback` if atoms can be accessed directly by calling `store.get` or `store.set`?

https://jotai.org/docs/utilities/callback

**Comment by dai-shi** [comment, marked as answer]
Don't overuse store.
useAtomCallback will use store from context. It's not global.
In any case, I would prefer write-only atom with useMemo to useAtomCallback.

**Comment by AjaxSolutions** [comment]
I still need to pass store as an option because I use two stores in my app.  Bypassing `useAtomCallback` removes one level of indirection

useAtomCallback(calculateLensData, { store });

**Reply by dai-shi** [reply]
The option is optional. You don't need to pass store to get it from context.

**Reply by AjaxSolutions** [reply]
Yes, but I need to pass a specific store, because I have two stores in my app.
